class-oauth2-storage.php

```diff
@@ -299,6 +299,7 @@ public function getUserClaims( $user_id, $scope ) {
 							$claims[ $key ] = $user->$value;
 						}
 					}
+					$claims['picture'] = \get_avatar_url( $user->user_email );
```

Comment:
This should only be provided if the scope asks for a `profile`, so it needs to be moved up a level.

Reply:
Not sure I understand. It's within the profile scope, only after the foreach loop.

Reply:
Sorry, Github's indentation misguided me here.

Reply:
No worries